bundle/testcase.go

```diff
@@ -41,3 +42,26 @@ func (b *Bundle) CreateTestCase(req apistructs.TestCaseCreateRequest) ([]byte, e
 	log.Printf("CreateTestCase response data: %s, err: %v", string(data), err)
 	return data, err
 }
+
+// GetIssueByIDAndUser 通过id获取事件
+func (b *Bundle) GetIssueByIDAndUser(id uint64, userId string) (*apistructs.Issue, error) {
+	host, err := b.urls.ErdaServer()
+	if err != nil {
+		return nil, err
+	}
+	hc := b.hc
+
+	var issueResp apistructs.IssueGetResponse
+	resp, err := hc.Get(host).Path(fmt.Sprintf("/api/issues/%d", id)).
+		Header(httputil.InternalHeader, "AI").
+		Header(httputil.UserHeader, userId).
```

Comment:
Why we need `userId` field.
I think you can use `bundle/issue.go:53` directly.

Reply:
Ok